CancerGov/_src/StyleSheets/_searchresults.scss

```diff
@@ -16,6 +16,13 @@
 	dd {
 		margin-bottom: 1em;
 	}
+	.dictionary-list {

```

Comment:
This probably does not need to be nested inside of .results. That makes it very specific.